### PR TITLE
Switch to using a light theme on home page, fixes loading issues

### PR DESCRIPTION
**Pull request by conrad-vanl**
Screen Shot 2019-10-09 at 9 23 20 AM

## TODO:

- [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] Closes [tag-issues-here]
- [ ] No new warnings in tests, in storybook, and in-app
- [ ] Upload GIF(s) of iOS and Android
- [ ] Set a relevant reviewer

## REVIEW:

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

**Comment by vinnyjth** [comment]
image

**Comment by vinnyjth** [comment]
@burkeshartsis can you give this a PR a spin locally, just so we get more than my eyes on it?

File: packages/WillowCreekApp/src/tabs/home/Home.js [hunk 1]
```diff
@@ -117,101 +114,87 @@ class Home extends PureComponent {
...
-          const featuredColor = get(featuredItem, 'theme.colors.primary')
-            ? Color(get(featuredItem, 'theme.colors.primary'))
-                .darken(0.5)
-                .toString()
-            : null;
+          //           const featuredColor = get(featuredItem, 'theme.colors.primary')
+          //             ? Color(get(featuredItem, 'theme.colors.primary'))
+          //                 .darken(0.5)
+          //                 .toString()
+          //             : null;
```
[the remark this answers is not in the document]
Comment (burkeshartsis) [reply packages/WillowCreekApp/src/tabs/home/Home.js]: Lines 5 (`color`)  and 32-39 (`BackgroundFill`) are also dead code

**Comment by burkeshartsis** [comment]
updated screenshots with my `StatusBar` fix

File: packages/WillowCreekApp/src/tabs/home/Home.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (burkeshartsis) [reply packages/WillowCreekApp/src/tabs/home/Home.js]: fixed in 89da4ae
